sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosQueryRequestOptions.java

```diff
@@ -28,6 +28,7 @@
      * Instantiates a new query request options.
      */
     public CosmosQueryRequestOptions() {
+        this.queryMetricsEnabled = true;
```

Comment:
how about setting on variable declaration itself

Reply:
Looks like the most common pattern has been to initialize in the ctor if a ctor is present - CosmosDatabaseProperties, CosmosTriggerProperties etc. - so followed that approach for consistency.